.gitignore

```diff
@@ -1,4 +1,5 @@
 /gatsby-ssr.js
 /gatsby-browser.js
+/antiflicker.js
```

Comment:
why is it in gitignore?

Reply:
I assumed similarly to gatsby-ssr and gatsby-browser, as they are assets produced by the build they should be ignored